package.json

```diff
@@ -6,10 +6,20 @@
 	"publisher": "eg2",
 	"icon": "npm_icon.png",
 	"engines": {
-		"vscode": "^1.44.0"
+		"vscode": "^1.66.0"
 	},
-	"workspaceTrust": {
-		"request": "never"
+	"capabilities": {
+		"virtualWorkspaces": {
+			"supported": false,
+			"description": "Npm scripts running is not possible in virtual workspaces."
+		},
+		"untrustedWorkspaces": {
+			"supported": false,
+			"description": "The extension requires workspace trust because it runs npm scripts from the workspace.",
+			"restrictedConfigurations": [
```

Comment:
@egamma, `restrictedConfigurations` should be used only in the scenario in which an extension has `limited` support in an untrusted workspace. Since this extension does not get activated in an untrusted workspace there is no need to make changes to the settings declaration.